runners/spark/src/main/java/org/apache/beam/runners/spark/util/BroadcastHelper.java

```diff
@@ -31,97 +30,53 @@
 /**
  * Broadcast helper.
  */
-public abstract class BroadcastHelper<T> implements Serializable {
+public class BroadcastHelper<T> implements Serializable {
```

Comment:
Maybe this should not be called `BroadcastHelper` anymore... `SideInputBroadcast` ?